src/Math-Polynomials/PMPolynomial.class.st

```diff
@@ -26,6 +26,15 @@ PMPolynomial class >> coefficients: anArray [
 	^self new initialize: anArray reverse
 ]
 
+{ #category : #information }
```

Comment:
I'm not sure I would put this method on the class side but I could understand. 
Now why the PMNewtonZeroFinder does not define a creation class method  desiredPrecision: like that you do not need this method and the class PMNewtonZeroFinder gets improved.

Reply:
I saw that you did it :)

Reply:
Yes, I had to take smaller steps in order to keep our tests green.